NFC_EEPROM/TARGET_MM/PinNames.h

```diff
@@ -0,0 +1,343 @@
+/* 
```

Comment:
It doesn't seem right to have that file in that PR. `PinNames.h` should be provided by the mbed os target.

Reply:
this is custom target not meant to be in mbed-os

Reply:
It has not place here.